documentation/manual/src/main/asciidoc/en-US/modules/redis.asciidoc

```diff
@@ -1196,11 +1197,11 @@ Key: User:user_001
 { 
   "addresses" : [
     { 
-      "addresses_KEY" : "work",
+      "addressType" : "work",
```

Comment:
Adjusted the example using the resulting JSON of `org.hibernate.ogm.datastore.redis.test.associations.MapMappingTest`